tests/sentry/auth/test_superuser.py

```diff
@@ -189,6 +193,30 @@ def test_su_access_no_request(self, logger):
                 "superuser.logged-in", extra={"ip_address": "127.0.0.1", "user_id": 10}
             )
 
+    @freeze_time("2022-03-03 08:21:34")
```

Comment:
could you reconstruct the arguments to freeze_time as the base time + a timedelta? it is hard for me to follow otherwise.